portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,155 @@
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>hey, it's malachi 🚀</title>
+        <!-- <link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css"> -->
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+        <link rel="stylesheet" href="style.css">
+        <script src="https://use.fontawesome.com/3e86b2749f.js"></script>
+        <link rel="shortcut icon" href="images/moon.png"/>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+        <!-- <script src="https://unpkg.com/aos@next/dist/aos.js"></script> -->
+        <script src="script.js"></script>
+    </head>
+    <body>
+        <!-- HEADER -->
+        <header>
+            <nav class="top-nav">
+                <ul class="nav-list">
+                    <li>
+                        <a href="index.html" class="nav-link">home</a>
+                    </li>
+                    <li>
+                        <a href="#biography" class="nav-link">biography</a>
+                    </li>
+                    <li>
+                        <a href="#projects" class="nav-link">projects</a>
+                    </li>
+                     <li>
+                        <a href="https://drive.google.com/file/d/1KUyDlDXmT-jwwOFXAHw0-48sdiVLewFJ/view" class="nav-link">resume</a>
+                    </li>
+                     <li>
+                        <a href="http://linkedin.com/in/malachiblenman" class="nav-link">linkedin</a>
+                    </li>
+                     <li>
+                        <a href="http://github.com/malachiblenman" class="nav-link">github</a>
+                    </li>
+                     <li>
+                        <a href="https://www.instagram.com/mallyblenman/" class="nav-link">instagram</a>
+                    </li>
+                </ul>
+            </nav>
+            <div class= "about-me-pic"> 
+                <img id="selfie" src="images/mally2.JPG">
+                 <img id="moon" src="images/moon.png">
+                </div>
+            <div class="landing-text">
+              <h1>malachi blenman
+                  <p>computer engineering major @ binghamton university</p>
+                  <p>aspiring aerospace engineer, graphic designer and social media influencer</p></h1>
+
+            </div>
+        </header>
+
+<!-- BIOGRAPHY -->
+        <section class="biography" id="biography">
+            <div class="container">
+                <div class="section-heading">
+                  <img id = "biographypic" src = "images/bio">
+                </div>
+
+                <div class="portfolio-item">                    
+                    <div class="portfolio-description">
+                        <img src = "images/aboutme.png">
```

Comment:
nit: drop the spaces around the `=` for consistency with the rest of the file.

Reply:
Done!